vassal-app/src/main/java/VASSAL/build/module/turn/TurnTracker.java

```diff
@@ -167,6 +167,7 @@ public class TurnTracker extends TurnComponent implements CommandEncoder, GameCo
 
   protected int plusButtonWidth  = DEFAULT_SIZE;
   protected int turnButtonHeight = DEFAULT_SIZE;
+  protected Boolean fwdOnly = false;
```

Comment:
This should be a `boolean` instead, as you can take advantage of auto-unboxing.

Reply:
doPrev() still has "if" statement to block hotkey activation.